### PR TITLE
Fix for displayName transform

### PR DESCRIPTION
**Pull request by kassens**
The `react.displayName` transform was added in 93b9329b758cde3e921b26e11ba91d9700d2a06d.

That diff missed to update the `.babelrc` where the comment says it should stay
in sync (I'm not sure where it's used though). I added a comment in the other
direction so this can be prevented in the future.

I also updated the `cacheVersion` so we actually transform the code again to add
the missing displayName properties to unchanged components.

### CODE REVIEW

**Comment by ide** [comment]
LGTM -- @kassens can you summon the bot to import this?

File: local-cli/bundle.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ function getBundle(flags) {
     projectRoots: projectRoots,
     transformModulePath: require.resolve('../packager/transformer.js'),
     assetRoots: assetRoots,
-    cacheVersion: '2',
+    cacheVersion: '3',
```
Comment (vjeux) [inline local-cli/bundle.js]: This file is not synced internally. Can you submit a different pull request for this one?

**Comment by kassens** [comment]
@facebook-github-bot import
